src/shared/treePaths/useTreePaths.test.tsx

```diff
@@ -70,7 +71,11 @@ const overviewMock = {
 }
 
 describe('useTreePaths', () => {
-  function setup({ repoOverviewData }) {
+  function setup({
+    repoOverviewData,
+  }: {
+    repoOverviewData: typeof overviewMock
```

Comment:
this would actually be the type of the zod schema right? 
https://github.com/codecov/gazebo/blob/68a1e3a485c8c8b97783e8b1a15747f9f4dc0fc1/src/services/repo/useRepoOverview.tsx#L9

Also if the mock isn't changing, could modify line 81 to just use the mock directly and remove the need to pass it in outright